xbmc/dialogs/GUIDialogYesNo.cpp

```diff
@@ -65,12 +74,16 @@ bool CGUIDialogYesNo::OnBack(int actionID)
 {
   m_bCanceled = true;
   m_bConfirmed = false;
+  m_bCustomClicked = false;
   return CGUIDialogBoxBase::OnBack(actionID);
 }
 
 void CGUIDialogYesNo::OnInitWindow()
 {
-  SET_CONTROL_HIDDEN(CONTROL_CUSTOM_BUTTON);
+  if (m_bCustomEnabled)
```

Comment:
Maybe named a little bad? Enabled is something that can be disabled for me. This seems to set visibility, so shouldn't it be `m_bCustomVisible`?